src/kapla/projects/base.py

```diff
@@ -140,10 +141,9 @@ class BasePythonProject(BaseProject[SpecT]):
 
     _python_executable: Path
 
-    @property
-    def venv_path(self) -> Path:
-        """Return path to virtualenv root directory"""
-        return self.root / ".venv"
+    def __init__(self, filepath: Union[str, Path], venv_path: str = None):
```

Comment:
```suggestion
    def __init__(self, filepath: Union[str, Path], venv_path: Optional[str] = None):
```

Reply:
Also good catch ;) Because `no_implicit_optional` is set to `True` in [config](https://github.com/charbonnierg/kapla-v2/blob/7283889da1a7d46721b65101528db58b1c05604a/setup.cfg#L50), we need to explicitely type argument as optional